specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json

```diff
@@ -0,0 +1,1563 @@
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "An array of image templates",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ImageTemplate"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The continuation token."
+        }
+      }
+    },
+    "ImageTemplateSource": {
+      "type": "object",
+      "description": "Describes a virtual machine image source for building, customizing and distributing",
+      "discriminator": "type",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Specifies the type of source image you want to start with."
+        }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "ImageTemplatePlatformImageSource": {
+      "type": "object",
+      "description": "Describes an image source from [Azure Gallery Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages).",
+      "x-ms-discriminator-value": "PlatformImage",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ImageTemplateSource"
+        }
+      ],
+      "properties": {
+        "publisher": {
+          "type": "string",
+          "description": "Image Publisher in [Azure Gallery Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages)."
+        },
+        "offer": {
+          "type": "string",
+          "description": "Image offer from the [Azure Gallery Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages)."
+        },
+        "sku": {
+          "type": "string",
+          "description": "Image sku from the [Azure Gallery Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages)."
+        },
+        "version": {
+          "type": "string",
+          "description": "Image version from the [Azure Gallery Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages). If 'latest' is specified here, the version is evaluated when the image build takes place, not when the template is submitted."
+        },
+        "exactVersion": {
+          "type": "string",
+          "description": "Image version from the [Azure Gallery Images](https://docs.microsoft.com/en-us/rest/api/compute/virtualmachineimages). This readonly field differs from 'version', only if the value specified in 'version' field is 'latest'.",
+          "readOnly": true
+        },
+        "planInfo": {
+          "$ref": "#/definitions/PlatformImagePurchasePlan",
+          "description": "Optional configuration of purchase plan for platform image."
+        }
+      }
+    },
+    "ImageTemplateManagedImageSource": {
+      "type": "object",
+      "description": "Describes an image source that is a managed image in customer subscription. This image must reside in the same subscription and region as the Image Builder template.",
```

Comment:
>This image must reside in the same subscription and region  [](http://example.com/codeflow?start=98&length=59)

Is this a new restriction being introduced with this API version or is this just an update to the description?

Reply:
This was always a requirement that was enforced but not documented here. We are just clarifying the documentation here.